src/Debug/TraceableHub.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Mercure Component project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Mercure\Debug;
+
+use Symfony\Component\Mercure\HubInterface;
+use Symfony\Component\Mercure\Jwt\TokenFactoryInterface;
+use Symfony\Component\Mercure\Jwt\TokenProviderInterface;
+use Symfony\Component\Mercure\Update;
+use Symfony\Component\Stopwatch\Stopwatch;
+use Symfony\Contracts\Service\ResetInterface;
+
+/**
+ * Traces updates for profiler.
+ *
+ * @author Saif Eddin Gmati <azjezz@protonmail.com>
```

Comment:
Since most of the logic has been copied from the deprecated class, I think preserving credits from the deprecated class in addition to adding yourself as author would be fair :)